tests/test.js

```diff
@@ -1,8 +1,53 @@
 import test from 'tape';
+import data from '../src/fixtures/data';
+import calculator from '../src/lib/calculator';
 
-test('1 = 1 ?', (t) => {
-  const actual = 1;
-  const expected = 1;
-  t.equal(actual, expected, 'Should return 1 when given 1');
+test('Test #0', (t) => {
+  const expected = {
+    liters: 9,
+    cost: 0.0801,
+  };
+
+  const actual = calculator(data, { toilet: 2 });
+  t.deepEqual(actual, expected);
```

Comment:
You should have a third argument here which is a string. This should be a descriptive error message that shows up when this test fails.

Reply:
This applies to all tests below :point_down: